wsinfer/write_geojson.py

```diff
@@ -64,62 +66,52 @@ def _dataframe_to_geojson(df: pd.DataFrame, prob_cols: list[str]) -> dict:
     }
 
 
-def convert(input: str | Path, output: str | Path) -> None:
-    df = pd.read_csv(input)
+def make_geojson(results_dir: Path, csv: Path) -> None:
+    filename = csv.name.split(".")[0]
+    df = pd.read_csv(csv)
     prob_cols = [col for col in df.columns.tolist() if col.startswith("prob_")]
     if not prob_cols:
         raise click.ClickException("Did not find any columns with prob_ prefix.")
     geojson = _dataframe_to_geojson(df, prob_cols)
-    with open(output, "w") as f:
+    with open(f"{results_dir}/model-outputs-geojson/{filename}.json", "w") as f:
         json.dump(geojson, f)
 
 
-@click.command()
-@click.argument(
-    "results_dir",
-    type=click.Path(
-        exists=True, file_okay=False, dir_okay=True, path_type=Path, resolve_path=True
-    ),
-)
-@click.argument(
-    "output",
-    type=click.Path(exists=False, path_type=Path, resolve_path=True),
-)
-def togeojson(*, results_dir: Path, output: Path) -> None:
-    """Convert model outputs to GeoJSON format.
+def parallelize_geojson(csvs: list, results_dir: Path, num_workers: int) -> None:
```

Comment:
can we rename this to `write_geojsons`? the parallelization is an implementation detail that doesn't have to be referenced in the name. the more important feature is that it writes the files.

can you also clarify the type for `csvs`? instead of `list`, i believe it should be `list[Path]`.

Reply:
Sure. It is indeed a list of Path objects

Reply:
yes the object `csvs` will be a list, but it's also useful to type-hint what the list contains. type checkers can test if a non-allowed type is in the list, like a string for example.